src/Facade/Services/StoresPalletFacadeService.cs

```diff
@@ -120,7 +120,9 @@ protected override async Task UpdateFromResourceAsync(
 
         protected override Expression<Func<StoresPallet, bool>> SearchExpression(string searchTerm)
         {
-            throw new NotImplementedException();
+            bool isInt = int.TryParse(searchTerm, out int palletNumber);
+
+            return x => isInt && x.PalletNumber == palletNumber;
```

Comment:
this is just a FindById now? So no need to support an additional search endpoint really? I would just call findbyid wherever you need it